### PR TITLE
Core & Internals: Move FTSState & FTSCompleteStatus to common/constants Fix #4241

### LINKED ISSUE #4241: Replace FTSState
Motivation
----------

FTSState is not a database column, but still in `db/constants.py`, that's why it has to be used with `.name` all over the code.

Modification
------------

Migrate from `db/constants.py` to `common/constants.py` and replace the appropriate code parts with better checks.

### PR DESCRIPTION
**Pull request by sahandilshan**
Core & Internals: Move FTSState & FTSCompleteStatus to common/constants Fix #4241
------------------
This PR Fix #4241 by moving FTSSate & FTSCompleteState to common/constants.py

### CODE REVIEW

File: lib/rucio/common/constants.py [hunk 1]
```diff
@@ -33,3 +35,10 @@
               's3': ['https', 'davs', 's3']}
 
 SUPPORTED_PROTOCOLS = ['gsiftp', 'srm', 'root', 'davs', 'http', 'https', 'file', 's3', 's3+rucio', 's3+https', 'storm']
+
+FTS_STATE_NAMED_TUPLE = namedtuple('FTS_STATE', ['SUBMITTED', 'READY', 'ACTIVE', 'FAILED', 'FINISHED', 'FINISHEDDIRTY',
+                                                 'CANCELED'])
+FTS_STATE = FTS_STATE_NAMED_TUPLE('SUBMITTED', 'READY', 'ACTIVE', 'FAILED', 'FINISHED', 'FINISHEDDIRTY', 'CANCELED')
```
Comment (mlassnig) [inline lib/rucio/common/constants.py]: what am I missing here that you do the two steps with the named tuple? can't you just contract it?

Comment (sahandilshan) [reply lib/rucio/common/constants.py]: It can be done in one line. I thought this way is more clearer. Is it okay to do it in one line?

Comment (mlassnig) [reply lib/rucio/common/constants.py]: I think it's fine to contract. It's not overly complicated and constrained to this file. Just play with the indents to make it obvious.

Comment (sahandilshan) [reply lib/rucio/common/constants.py]: Hi @mlassnig. The request changes have been made.

**Review by mlassnig — APPROVED** [review]

**Comment by bari12** [comment]
Merging this as a feature for 1.26.

**Pull request merged**